src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Startup.cs

```diff
@@ -117,7 +117,16 @@ public override void ConfigureServices(IServiceCollection services)
                 var client = new ElasticClient(settings);
                 services.AddSingleton<IElasticClient>(client);
                 services.Configure<ElasticOptions>(o =>
-                    o.Analyzers.Add(new ElasticAnalyzer(ElasticSettings.StandardAnalyzer, new StandardAnalyzer())));
```

Comment:
By default we should only register the StandardAnalyzer.

Reply:
@Skrypt Why you said we should only register the standard analyzer? Is there an issue from registering all the built-in analyzers? The only analyzer that should be added when needed is the custom one, the rest should be available since ES supports them.

Reply:
Because the idea is to be able to create your own custom analyzers too. We do the same with Lucene where we only register the standard one. This way we leave the choice to people to register analyzers in a custom module. Here, the ElasticSettings should have an Analyzer setting that defaults to the standard one when none is specified. Then you can extend the number of analyzers available through a custom module extension. Because these other analyzers are really specific. We don't want to keep a list of analyzers in the ElasticSettings either because it will not reflect what it should do. The list should be retrievable by listing the Analyzers registered on that ElasticSearch instance.

Reply:
See this here: https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-custom-analyzer.html

Reply:
I don't think I follow you. Analyzers are registered in the IoC container not in the settings. The user still have full control because in each index added they can select which analyzer to apply during the search.

I don't see a valid reason not to provide this function out of the box since ES completely support it

Reply:
Because these analyzers are there to be used for building custom ones. You don't register them one by one normally.

Reply:
Only custom analyzers should be added as needed because you'll have to add your own rules to create the analyzer.

However, supported analyzers are great add to allow the user to chose from. Standard analyzer isn't any different that stopwords from OC prospective.

Reply:
hmm, stopword is an option on each analyzer that can be configured. The standard analyzer should be pretty much the same as the Lucene one. The difference is that Lucene provides some predefined Analyzers for some specific cultures.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-analyzers.html

These are ES supported analyzers than can be used when running a query. This will change how to term in analyzed by ES. So when we say use "Stopword" or any of the built-in analyzers, the ES will know how to handle each. However, if someone want to define a custom analyzer, we create that analyzer when we create the index 
https://github.com/OrchardCMS/OrchardCore/blob/0ab72a206f613839ec28fbd588bee0f7fa2e3fd8/src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Services/ElasticIndexManager.cs#L85